sdk/storage/azblob/blob/client.go

```diff
@@ -624,7 +626,9 @@ func (b *Client) DownloadFile(ctx context.Context, file *os.File, o *DownloadFil
 	}
 
 	if size > 0 {
-		return b.downloadFile(ctx, file, *do)
+		writeSize, err := b.downloadFile(ctx, file, *do)
+		_, _ = file.Seek(filePointer, 0)
```

Comment:
Add error check here. We can also do something like,
```suggestion
		_, _ = file.Seek(filePointer, io.SeekStart)
```